app/views/categories/show.html.erb

```diff
@@ -0,0 +1,28 @@
+<%= stylesheet_link_tag 'category_index' %>
+<% content_for :page_name, "Transactions" %>
+
+<%= render 'helper/navbar' %>
+<div class="px-4 d-flex justify-content-between align-items-center top-card">
+  <h2 class="second-color"><%= @category.name %></h2>
+  <h2 class="second-color">$ <%= @category.total %></h2>
+</div>
+<div class="container-fluid">
+  <% @category.buys.order(created_at: :desc).each do |buy| %>
+    <div class="card mt-3 px-3">
+      <div class="card-body d-flex justify-content-between">
+        <div class="d-flex align-items-center">
+          <h2 class="text-decoration-none second-color"><%= buy.name %></h2>
+        </div>
+        <h3 class="second-color align-self-center" >$ <%= buy.amount %></h3>
+      </div>
+    </div>
+  <% end %>
+
+  <div class="d-flex justify-content-center mt-4">
+    <%= link_to 'Add New Transaction', new_buy_path, id: 'add-transaction', class: 'btn btn-primary', style: "width: 100%;" %>
+  </div>
+  <div class="d-flex justify-content-center mt-4">
+    <%= link_to 'Back', homepage_path, class: 'btn btn-warning', style: "width: 100%;" %>
+  </div>
```

Comment:
- You have done well so far, but please make sure your back (<) button is designed and placed as it is placed in the design template. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏. Please fix this 👍 👍

Your project:
![rgergwer](https://github.com/AliRezaBmeDu/budget-app/assets/73607512/0586aa1e-0cb8-4bcf-b3fe-7612322e174e)

Design template:
![bajlntshaet](https://github.com/AliRezaBmeDu/budget-app/assets/73607512/0928764f-4c9d-4e20-82f1-61d6bd654c9c)